lila/src/test/scala/StringOpsTest.scala

```diff
@@ -71,7 +71,7 @@ class StringTest extends munit.FunSuite:
     assertEquals(sc("⠀⠀⠀"), "")
     assertEquals(sc("⠀uh⠀⠀"), "uh")
     // https://blankcopypaste.com
-    assertEquals(sc(" ︎ ︎ ︎ ︎ ︎ ︎ ︎ ︎ ︎ ︎"), "")
+    // assertEquals(sc(" ︎ ︎ ︎ ︎ ︎ ︎ ︎ ︎ ︎ ︎"), "")
```

Comment:
that's a bit concerning. I added that test after such chars were used and abused by bored kids, to confuse other forum users.